java-checks/src/test/files/checks/SimpleClassNameCheck.java

```diff
@@ -0,0 +1,29 @@
+import java.util.*;
+import com.google.common.collect.*;
+import com.google.common.collect.ImmutableList;
+import java.awt.*;
+import java.nio.charset.*;
+import java.nio.*;
+
+class A {
+
+  void notWildcardImport() {
+    com.google.common.collect.ImmutableList list;      // Noncompliant
+    com.google.common.collect.ImmutableList.Builder<Object> builder =  // Noncompliant [[startColumn=5;endColumn=44]]
+      com.google.common.collect.ImmutableList.builder(); // Noncompliant {{Replace this fully qualified name with "ImmutableList"}}
+    System.out.println(com.google.common.collect.ImmutableList.class); // Noncompliant [[startColumn=24;endColumn=63]]
+
+    ImmutableList.builder();
+    ImmutableList anotherList;
+  }
+
+  void wildcardImport() {
+    java.util.List<String> myList =      // Noncompliant {{Replace this fully qualified name with "List"}}
```

Comment:
This one is a FP... And I have no idea how we could avoid it.

If we remove `java.util.List`, the code won't compile, because of the ambiguity with `java.awt.List`.